scripts/install.sh

```diff
@@ -325,16 +325,17 @@ if [[ $enable_self_signed_ca == "true" ]]; then
   # comment out vars
   configMapGeneratorStartLine=$(grep -n configMapGenerator ./default/kustomization.yaml |cut -d':' -f1)
   configMapGeneratorBeforeLine=$((configMapGeneratorStartLine-1))
-  sed "1,${configMapGeneratorBeforeLine}s/^/#/g" -i default/kustomization.yaml
-  
+  sed -i.bak "1,${configMapGeneratorBeforeLine}s/^/#/g" default/kustomization.yaml
+
   # remove webhookcainjection_patch.yaml
-  sed 's+- webhookcainjection_patch.yaml++g' -i default/kustomization.yaml
+  sed -i.bak  's+- webhookcainjection_patch.yaml++g' default/kustomization.yaml
```

Comment:
Another fix we might need here: It looks like this line is attempting to remove the line
```
  - webhookcainjection_patch.yaml
```
but in the current version of `default/kustomization.yaml` the line is now 
```
  - path: webhookcainjection_patch.yaml
```
[REF](https://github.com/kserve/modelmesh-serving/blob/dd0229fe52d6c90703eabeecfdee287dd3ac55fd/config/default/kustomization.yaml#L65)

🤔 
I think we can just change this to a sed invocation that deletes a matching line instead of a replacement:
```
sed -i.bak '/webhookcainjection_patch.yaml/d' default/kustomization.yaml
```

Reply:
Yes I agree with travis. We can use the sed command to remove the line.

Reply:
Just for reference, I commented that the -i.bak is for Mac os version option.